Nguyen_Ken/Assignments/full_friends/templates/delete.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang='en'>
+<head>
+	<title>Full Friends Delete</title>
+	<meta charset='UTF-8'>
+	<meta name='description' content='Ninja Gold'>
+	<link rel="stylesheet" type="text/css" href="{{url_for('static', filename='css/styles.css')}}">
+</head>
+<body>
+	<div class='wrapper'>
+		<h1>Are You Sure You Want To Delete Your Friend?</h1>
+
+		{% for friend in oneFriend %}
+		<p>ID: {{friend['id']}}</p>
```

Comment:
My preference is not to display this to users, since they arent interested in it.